midpoint/quickstart/index.adoc

```diff
@@ -25,8 +24,8 @@ Before you proceed further, make sure the following requirements are satisfied o
 
 === Docker
 
-You need to have Docker installed on your computer to follow through with this guide.
-Check the link:https://docs.docker.com/engine/install/[official Docker documentation] for guides on installing Docker.
+You need to have Docker installed on your computer.
+Check the documentation on link:https://docs.docker.com/engine/install/[Docker Engine] and link:https://docs.docker.com/desktop/setup/install/windows-install/[Docker Desktop] for guides on setting up Docker.
```

Comment:
Installation of docker desktop for Windows is not relevant since this script is only for Linux and Mac.